### ISSUE TITLE
Quiz review shows wrong calculation. 

### ISSUE THREAD

**Issue by JohannesStoehr** [ls1intum/Artemis#3876]
### Describe the bug

The review of a drag and drop quiz shows the wrong formula to the student for the score when some spots are correctly left blank.  

### To Reproduce

1. Create a drag and drop quiz where some spots are correctly empty 
2. Participate as a student and correctly leave them empty.
3. Review the score as a student after the deadline and check the explanation for the achieved score. 

### Expected behavior

A correct formula. 

### Screenshots

Will follow as soon as I‘m back home.

### What browsers are you seeing the problem on?

Safari

**Comment by FrankeLukas** [comment]
Is the formular really wrong? You had to map 7 items in total, just two of which had an empty mapping as correct solution?

**Comment by JohannesStoehr** [comment]
> Is the formular really wrong? You had to map 7 items in total, just two of which had an empty mapping as correct solution?
Yes, @krusche confirmed that the student should only get 1.2 points. This is the correct formula that is used to calculate the 1.2 points and should be displayed: 4/5*2 - 1/5*2 = 1,5 - 0,3 = 1,2
So empty gaps apparently don't give any points.

**Comment by krusche** [comment]
Yes the calculation is done based on the number of mappings, not based on the number spots
